documents/extension/funding/funding.go

```diff
@@ -0,0 +1,33 @@
+package funding
+
+import (
+	"reflect"
+
+	clientfundingpb "github.com/centrifuge/go-centrifuge/protobufs/gen/go/funding"
+)
+
+// Data is the default funding extension schema
+type Data struct {
+	FundingId             string `json:"funding_id,omitempty" attr:"string"`
+	Amount                string `json:"amount,omitempty" attr:"string"`
```

Comment:
this will be decimal

Reply:
Yes. I will change all the field type in the next commit.